src/bio/smd/smd_store.c

```diff
@@ -269,7 +269,8 @@ smd_store_open(char *fname)
 
 	if (smd_df->smd_version > SMD_DF_VERSION ||
 	    smd_df->smd_version < SMD_DF_VER_1) {
-		D_ERROR("Unsupported DF version %d\n", smd_df->smd_version);
+		D_CRIT("Unsupported DF version %d.  Supported range: %d-%d\n",
```

Comment:
ras isn't available at this point as it uses things like dss_rank_self and tls that are not yet initialized when this is called

Reply:
Some questions so I can work out what needs to be fixed (as requested by @johannlombardi ) to enable event to be raised here;
- couldn't you just pass in NULL for the rank, it's supposed to be optional?
- what gets called wrt tls when the event gets raised that is not yet initialised?

Reply:
If you pass NULL, the framework currently calls dss_self_rank()
I worked around that problem in a local patch but passing a "pre_init" flag but it then runs into other issues, primarily relating to tls.   None of it is likely insurmountable but didn't want to tie up this patch trying to fix it.